### ISSUE TITLE
[DetailsList][Accessibilty][Question] Details list header column with no icon and name , should have some way to read header text

### ISSUE THREAD

**Issue by aadhars** [OfficeDev/office-ui-fabric-react#2144]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes)

Requested priority: (Normal) 

Products/sites affected: (if applicable)

#### Describe the issue:

We are using a details List in which some column header does not have and icon and name. We want to screen reader to pronounce such column header. Is there a way to achieve this via props?
 One way I can think of is to use custom renderer but then the question is, should detail list provide first class support for such case.
